backend/middleware/validation.js

```
@@ -84,59 +84,56 @@ const authSchemas = {
+        "string.pattern.base":
+          "Password must contain at least one uppercase letter, lowercase letter, number, and special character",
       }),
-    role: Joi.string().valid('patient', 'doctor', 'nurse').default('patient'),
+    role: Joi.string().valid("patient", "doctor", "nurse").default("patient"),
     specialty: Joi.string()
       .max(100)
-      .when('role', {
-        is: Joi.valid('doctor', 'nurse'),
+      .when("role", {
+        is: Joi.valid("doctor", "nurse"),
         then: Joi.required(),
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)